src/transformers/generation/utils.py

```diff
@@ -1673,7 +1673,7 @@ def generate(
             )
 
         # 8. prepare distribution pre_processing samplers
-        logits_processor = self._get_logits_processor(
+        prepared_logits_processor = self._get_logits_processor(
```

Comment:
changed the name so we can keep a separation between the custom logits processors (`logits_processor`) and the custom + loaded logits processors (now `prepared_logits_processor`)

assisted generation should know which logits processors are custom